vizro-core/src/vizro/models/_components/card.py

```diff
@@ -34,15 +34,16 @@ class Card(VizroBaseModel):
 
     @_log_call
     def build(self):
-        text = dcc.Markdown(self.text, className="card_text", dangerously_allow_html=False, id=self.id)
-        card_content = (
-            dbc.NavLink(
-                text,
-                href=get_relative_path(self.href) if self.href.startswith("/") else self.href,
-                className="card-link",
+        text = dcc.Markdown(self.text, dangerously_allow_html=False, id=self.id)
+        card_content = dbc.CardBody(
```

Comment:
Suggest refactoring this to remove the change here and instead put the `dbc.CardBody` in the return statement just to make it a bit more readable (if I understood the change correctly):
```
return dbc.Card(dbc.CardBody(card_content), className=card_class, id=f"{self.id}_outer")
```

Reply:
Let me actually remove the `dbc.CardBody` altogether. I think the `card_content` can be inserted directly into the `dbc.Card` 👍